workshops/parallax_effect/README.md

```diff
@@ -0,0 +1,176 @@
+---
+name: 'Parallax Effect'
+description: 'Creating greater user experience on web using vanilla javascript'
+author: '@giridhar7632'
+---
+
+# Parallax Effect
+**using vanilla javaScript**
+
+
+One of the most popular effects of the modern web experience is the **parallax effect**. We will be building basic-starter parallax effect 
```

Comment:
Check the grammar / phrasing on this sentence

Reply:
I did my best. 
Please let me know, if there are any changes to be done

Reply:
Is the grammar okay?